zanata-war/src/main/webapp/WEB-INF/layout/delete_confirmation_modal.xhtml

```diff
@@ -0,0 +1,38 @@
+<ui:composition xmlns="http://www.w3.org/1999/xhtml"
+  xmlns:ui="http://java.sun.com/jsf/facelets"
+  xmlns:a4j="http://richfaces.org/a4j"
+  xmlns:s="http://jboss.org/schema/seam/taglib">
+
+
+
+  <div class="modal" id="delete-confirmation-modal" tabindex="-1"
+    role="dialog" aria-hidden="true">
+    <div class="modal__dialog l--constrain-medium">
+      <header class="modal__header">
+        <h2 class="modal__title">#{msgs['jsf.DeleteConfirmation']}</h2>
+        <button class="modal__close button--link" data-dismiss="modal"><i
+          class="i i--huge i--cancel"></i></button>
+      </header>
+
+      <div class="modal__content">
+        <div class="message message--highlight l--pad-h-1">
+          #{msgs['jsf.DeleteConfirmation.Message']}
+        </div>
+      </div>
+
+      <footer class="modal__footer l--pad-h-1 l--pad-v-half bg--highest">
+        <div class="g--tight">
+          <s:div styleClass="g__item txt--align-right"
+            id="delete-confirmation-buttons">
+            <button id="deleteButton" styleClass="button--primary l--push-h-half" jsfc="a4j:commandButton"
+              onclick="runDelete();zanata.modal.hide('#delete-confirmation-modal')" value="#{msgs['jsf.Delete']}"></button>
+
+            <button id="cancelDelete" styleClass="button--danger" jsfc="a4j:commandButton"
```

Comment:
The style of the cancel button should be a link. See `new_language_modal.xhtml` for both of the button style and position.